jmeter-tests/readme.md

```diff
@@ -0,0 +1,25 @@
+These JARs should be under lib/ext directory of jmeter:
+
+archaius-core-0.4.1.jar   (com.netflix.archaius:archaius-core)
+commons-configuration-1.8.jar (commong-configuration:commons-configuration)
+commons-lang-2.6.jar   (commons-lang:commons-lang)
+compress-lzf-1.0.3.jar   (com.ning:compress-lzf)
+guava-18.0.jar    (com.google.guava:guava)
+hystrix-core-1.4.7.jar  (com.netflix.hystrix:hystrix-core)
+hystrix-metrics-event-stream-1.4.7.jar (com.netflix.hystrix:hystrix-metrix-event-stream)
+hystrix-servo-metrics-publisher-1.4.7.jar  (com.netflix.hystrix:hystrix-servo-metrix-publisher)
+jackson-annotations-2.5.3.jar (com.fasterxml.jackson.core:jackson-annotations)
+jackson-core-2.5.3.jar (com.fasterxml.jackson.core:core)
+jackson-databind-2.5.3.jar  (com.fasterxml.jackson.core:jackson-databind)
+rxjava-1.0.10.jar  (io.reactivex:rxjava)
+servo-core-0.8.0.jar   (com.netflix.servo:servo-core)
+servo-graphite-0.8.0.jar  (com.netflix.servo:servo-graphite)
+servo-internal-0.8.0.jar   (com.netflix.servo:servo-internal)
+
+Lightblue jars:
+
+jmeter-tests-1.8.0-SNAPSHOT.jar
+lightblue-client-core-1.8.0-SNAPSHOT.jar
+lightblue-client-http-1.8.0-SNAPSHOT.jar
+lightblue-client-hystrix-1.8.0-SNAPSHOT.jar
+lightblue-core-hystrix-1.8.0-SNAPSHOT.jar
```

Comment:
I am worried that this readme is going to be hard to maintain. I know for sure the lightblue versions are already old. Is the intention that these are the minimum versions required?